crates/matrix-sdk/src/event_cache/linked_chunk.rs

```diff
@@ -40,20 +40,20 @@ pub enum LinkedChunkError {
 /// of a single one. A chunk has a maximum capacity of `CAPACITY`. Once a chunk
 /// is full, a new chunk is created. Not all chunks are necessarily entirely
 /// full.
-pub struct LinkedChunk<T, const CHUNK_CAPACITY: usize> {
+pub struct LinkedChunk<T, U, const CHUNK_CAPACITY: usize> {
```

Comment:
Could we give meaningful names to T and U, to make them easy to understand by just reading the code here? Nothing mandates one-letter names for generics in Rust :)